src/main/java/core/basesyntax/dao/DaoManufacturerImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Manufacturer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class DaoManufacturerImpl implements DaoManufacturer {
+    @Override
+    public Manufacturer create(Manufacturer item) {
+        Storage.manufacturerId++;
```

Comment:
Mentors told us that Storage must generate id, you can add method in storage that add id like a key and increace it, but it's up to you